content/news/052/index.md

```diff
@@ -70,6 +71,31 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Untitled Pixel Wizards Game][pixel-wizards]
+<video controls autoplay muted loop> <source type="video/mp4" src="untitled-pixel-wizards-game.mp4" onerror="parentNode.parentElement.innerText = 'Sorry, your browser does not support the video codec. Try a different browser!'"> />
+</video>
+_Enemies now perceive, pursue and attack.. and occasionally get burned to death._
+
+[![A Hound chases a player, and its corpse ragdolls after it is burned to death by a fireball](untitled-pixel-wizards-game.gif)][pixel-wizards]
+_Enemies now perceive, pursue and attack.. and occasionally get burned to death._
+
+[Untitled Pixel Wizards Game][pixel-wizards] is a local-multiplayer [Noita]-like platformer about
+killing baddies using spells powered by pixel physics. This month was focused on juicing up said baddies:
+
+- [Pew Pew Pew][pixel-wizards-update-1]: baddies learned to shoot at players.
+- [Hot Pursuit][pixel-wizards-update-2]: baddies also learned to chase players! (They're real smart.)
+- [Status Update][pixel-wizards-update-3]: physically-simulated pixels learn to burn & poison players & baddies.
+- [Ragdolls][pixel-wizards-update-4]: corpses of dead baddies learn to tumble around all realistic-like.
+- [Fiddling with Fire][pixel-wizards-update-5]: the fire mechanic figures out how to better burn baddies.
+
+[pixel-wizards]: https://slowrush.dev
+[Noita]: https://store.steampowered.com/app/881100/Noita/
+[pixel-wizards-update-1]: https://www.slowrush.dev/news/pew-pew/
+[pixel-wizards-update-2]: https://www.slowrush.dev/news/hot-pursuit/
+[pixel-wizards-update-3]: https://www.slowrush.dev/news/status-update/
+[pixel-wizards-update-4]: https://www.slowrush.dev/news/ragdolls/
+[pixel-wizards-update-5]: https://www.slowrush.dev/news/fiddling-with-fire/
```

Comment:
This update is not live yet - and so will fail CI for now - but it will be live by approximately 2024-06-28 11:00 UTC - most likely around 02:00 UTC, as that's when I normally hit publish. (I still need to work out a good workflow for automating un-drafting Zola content at specific times.)

I figured it was worth getting this PR up a little early in case you wanted to discuss the video thing.. and also submit my update a tiny bit earlier!

Reply:
Ok published this now, so CI is green now.